src/Columns.tsx

```diff
@@ -8,6 +8,7 @@ export const SelectColumn: Column<any, any> = {
   key: 'select-row',
   name: '',
   width: 35,
+  maxWidth: 35,
```

Comment:
`width` doesn't override `minColumnWidth` anymore, so we specify `maxWidth`.
`width` is just the default width.
We still specify `width` here so we properly calculate `unallocatedColumnWidth`.